_layouts/landing.html

```diff
@@ -20,3 +20,23 @@ <h1 class="post-title">{{ page.title }}</h1>
     {% include sidebar.html %}
   </div>
 </div>
+<script>
+  !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t){var e=document.createElement("script");e.type="text/javascript";e.async=!0;e.src=("https:"===document.location.protocol?"https://":"http://")+"cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var n=document.getElementsByTagName("script")[0];n.parentNode.insertBefore(e,n)};analytics.SNIPPET_VERSION="4.0.0";
+  analytics.load("GvscnIMX05fG8s2MEZY7PkSV4BTsFvOU");
+  analytics.page();
+  }}();
+  try {
```

Comment:
Is this landing being use currently?. I didn't see where

Reply:
Yes, for instance: 2017-01-01-dev-landing.markdown

Reply:
Right, I totally missed that. I think that instead of adding it in `landing` and `home` you can just add that in `default.html` which will be always included. Check here: https://github.com/play-with-docker/play-with-docker.github.io/blob/master/_layouts/default.html#L34

I'm just trying to find the better way so you don't have to copy/paste everywhere.

Reply:
The problem with adding it to default is then it shows up in post as well. Post waits until init is finished before running the same check.

Reply:
That would mean two checks on the same page

Reply:
You can skip that by adding a condition in the `default.html` like:
```
{% if page.layout != 'post' &}
// call analytics here. This way it won't run in the posts pages.
{% endif %}
```